website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

```diff
@@ -8,7 +8,15 @@ description: Understand Snowflake support for Apache Iceberg.
 dbt supports materializing the table in Iceberg table format in two different ways:
 
 - The model configuration field `table_format = 'iceberg'` (legacy)
-- Catalog integration in the model, resource, or dbt_project.yml configuration
+- Catalog integration can be configured in the SQL config (inside the `.sql` model file), property file (model folder), or project file ([`dbt_project.yml`](/reference/dbt_project.yml)).
+
+:::tip **Catalog integration configuration**
+
+You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or your `dbt_project.yml`.
```

Comment:
> You can also
I'm finding this note a little confusing.  On [this page](https://docs.getdbt.com/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) we say:

> 1. Create a catalogs.yml at the top level of your dbt project.
> 2. Apply the catalog configuration at either the model, folder, or project level.

This excerpt tells me that (at least for managed repos), you **need to** (1) create a catalogs.yml and also (2) apply the configs. But the note indicates it's optional? @amychen1776 you need to define the catalogs.yml, right?